pkgs/development/misc/jupyterKernels/default.nix

```diff
@@ -0,0 +1,66 @@
+{pkgs, python3, lib, kernelDefinitions ? null }:
+
```

Comment:
let
  defaultKernelDefinitions = {
    ...
  };
in {python3, lib, kernelDefinitions ? defaultKernelDefinitions }: